src/React/Widgets/NumberSliderWidget/index.js

```diff
@@ -62,14 +62,14 @@ export default React.createClass({
           className={style.range}
           value={this.props.value}
           onChange={this.valInput}
-          max={max} min={min}
+          max={max} min={min} step={this.props.step || 'any'}
         />
         <input
           type="number"
           className={style.text}
           value={this.props.value}
           onChange={this.valInput}
-          max={max} min={min}
+          max={max} min={min} step="any"
```

Comment:
I should test it in IE :) 
I didn't want the number input to turn red and complain if you enter a more precise number. However, it does affect what the arrows inside the text input do. I wish they could be controlled separately.